### ISSUE TITLE
Local Notification Undefined Behavior

### ISSUE THREAD

**Issue by kanielv** [ufosc/Alarm-Clock#74]
# Expected Behavior

When the user adds an alarm for a specific time, a local notification should appear saying the alarm went off. If the alarm is on repeat, then it should repeat again the next day at the same time. The user should also be able to disable this alarm.

# Current Behavior
Alarm notification pops up despite the alarm being turned off or deleted. The time when the alarm goes off is not accurate to the time set (off by an hour).

## Steps to Reproduce

Please provide detailed steps for reproducing the issue.

1. In the alarms tab set up an alarm (Set Alarm Time).
2. Configure an alarm and save it.

**Comment by carlsuarez** [comment]
Can I take up this issue?

**Comment by carlsuarez** [comment]
Could I work on this issue?

**Comment by kanielv** [comment]
> Could I work on this issue?

Sure!

**Comment by carlsuarez** [comment]
I fixed the issues with notifications triggering at the wrong time and the delete functionality. However, I'm having trouble with the "turn off" feature for alarms. Since Expo doesn't support rescheduling or disabling notifications, turning off an alarm would mean deleting it entirely, which would also remove the alarm card in the current implementation. Implementing a proper solution would require major code changes, and I don’t have time for that right now. If you'd like, I can open a PR for the issues I’ve already fixed.

**Comment by kanielv** [comment]
Of course. I'll take it a look at it and merge it if it looks good. I'll open another issue for properly deleting the an Alarm. Thanks for the help!
